src/coreclr/vm/amd64/AsmHelpers.asm

```diff
@@ -447,5 +449,13 @@ NESTED_END OnCallCountThresholdReachedStub, _TEXT
 
 endif ; FEATURE_TIERED_COMPILATION
 
-        end
+LEAF_ENTRY JIT_PollGC, _TEXT
```

Comment:
This is one of the APIs that also exists as asm in native AOT. One of the original goals here was to share assembly between CoreCLR and native AOT. Looking at the current native AOT implementation, there isn't much to obviously share, but perhaps the pattern could be started with this PR?

Reply:
I'd rather not start the pattern here. As you note, the implementations are actually somewhat different. I have work going on around Cached Interface Dispatch which I believe to be a better candidate for the sharing of code, which will better serve to setup a place to put shared code since it covers both C++ and asm bits that are being shared.

Reply:
Okay.